site/documentation/content/api/Metrics.md

```diff
@@ -75,6 +75,7 @@ Metrics provided by the protocol adapters are:
 | *hono.connections.authenticated*   | Gauge               | *host*, *component-type*, *component-name*, *tenant*                                         | Current number of connected, authenticated devices. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
 | *hono.connections.unauthenticated* | Gauge               | *host*, *component-type*, *component-name*                                                   | Current number of connected, unauthenticated devices. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
 | *hono.connections.authenticated.duration* | Timer        | *host*, *component-type*, *component-name*, *tenant*                                         | The overall amount of time that authenticated devices have been connected to protocol adapters. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
+| *hono.connections.attempts.total*  | Counter             | *host*, *component-type*, *component-name*, *outcome*                              | The number of connection attempts to protocol adapters. The outcome can be observed by looking the tag "outcome" which contains the result and the reason. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
```

Comment:
can you please replace the quotes around `"outcome"` with asterisks (`*outcome*`)?
It would also be helpful to define the possible values of the outcome tag in the corresponding table above.
FMPOV we just need `accepted`, `unauthorized`, `data-volume-exceeded`, `connection-count-exceeded` and `connection-duration-exceeded` for now ...

Reply:
Sure, no problem. 

Regarding the possible values: at the moment I only implemented one (see our discussion following your first comment) called "rejected-adapter-connection-limit-exceeded". I can rename this if you like but as far as I know there's also the possibility that a connection may be rejected because of a tenant specific connection limit, so I guess it makes sense to make this clear in the name.

Reply:
We could start with a single possible failure outcome for the time being. I see your point regarding the different types of limits. So maybe we can start with `adapter-connection-limit-exceeded`. WDYT? 
In any case, we should document the possible value(s) in the table ...

Reply:
Agreed and adapted.